mozilla-release/Jenkinsfile.android

```diff
@@ -0,0 +1,63 @@
+#!/bin/env groovy
+
+node('android') {
```

Comment:
Can you please change this line to:

node('us-east-1 && ubuntu && docker && !gpu') {